ui/js/pods-dfv/src/components/base-input.js

```diff
@@ -20,6 +20,8 @@ export const PodsDFVBaseInput = ( props ) => {
 			readOnly={ !!props.fieldConfig.readonly }
 			onChange={ props.onChange || handleChange }
 			onBlur={ props.onBlur }
+			min={ props.min }
```

Comment:
What if prop not defined?

Reply:
React will omit any attributes that are undefined or null